modules/mobile_session.lua

```diff
@@ -148,12 +155,18 @@ function mt.__index:Send(message)
         binaryData = message.binaryData
       }
     })
-  if not self.cor_id_func_map[self.correlationId] then
-    for fname, fid in pairs(functionId) do
-      if fid == message.rpcFunctionId then
-        self.cor_id_func_map[self.correlationId] = fname
-        break
+  -- During StartSession Send called with no correllation id
+  if message_correlation_id then
```

Comment:
@VVeremjova comment and if removed in https://github.com/AMikhniev/sdl_atf/commit/78c1c75063563f13dcaa3956e9dc1709d4e3f282. `message_correlation_id ` can't be `nil` ft this point.